scripts/swupd-performance/data_processing_script.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python3
+"""
+This module is created to clean, parse output of swupd update service
+and create a CSV file
+"""
+
+import argparse
+import os
+import json
+#from collections import OrderedDict
+import csv
+import logging
+from datetime import datetime
+import fileinput
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+JSON_KEYWORD1 = ["changed bundles", "new bundles", "deleted bundles",
+                 "changed files", "new files", "deleted files",
+                 "Statistics for going from version", "timestamp"]
+
+FILTER1 = ["info"]
+
+STATE_FILE = ".state"
+
+# keyword_list3 = ['Total execution time']
```

Comment:
another leftover

Reply:
So We in future might need this to calculate total execution time..Hence i kept it there along with the WIP function